OtripleS.Web.Api.Tests.Unit/Services/Foundations/Classrooms/ClassroomServiceTests.Exceptions.Remove.cs

```diff
@@ -85,7 +86,7 @@ public async Task ShouldThrowDependencyExceptionOnDeleteWhenDbExceptionOccursAnd
             this.loggingBrokerMock.Verify(broker =>
                 broker.LogError(It.Is(SameExceptionAs(
                     expectedClassroomDependencyException))),
-                    Times.Once);
+                        Times.Once);
```

Comment:
Remove the extra space here - just a tab.

Reply:
I was able to fix the indentation for this file.  The link for the updated is posted here: https://github.com/hassanhabib/OtripleS/pull/1550